### PR TITLE
Epbs payload

### PR DESCRIPTION
**Pull request by potuz**
This deals with sending FCU on the ePBS paths

- Removes calls from process CL blocks
- Adds a call after processing the payload
- Adds calls on reorgs by attestations
- Adds a call on late block tasks
- Also recreates the ticker for late block tasks to happen at 3 seconds instead of 4. 

Still broken by stategen (that needs to recover the right head state empty/full) and possibly the NSC

### CODE REVIEW

File: beacon-chain/blockchain/process_block.go [hunk 1]
```diff
@@ -483,9 +483,16 @@ func (s *Service) runLateBlockTasks() {
...
+	epbs := params.BeaconConfig().EPBSForkEpoch
 	for {
 		select {
-		case <-ticker.C():
+		case slot := <-ticker.C():
+			if slots.ToEpoch(slot) == epbs && slot%32 == 0 {
+				ticker.Done()
+				attThreshold := params.BeaconConfig().SecondsPerSlot / 4
+				ticker = slots.NewSlotTickerWithOffset(s.genesisTime, time.Duration(attThreshold)*time.Second, params.BeaconConfig().SecondsPerSlot)
+				time.Sleep(1 * time.Second)
```
Comment (terencechain) [inline beacon-chain/blockchain/process_block.go:494]: why sleep here?

File: beacon-chain/blockchain/process_block.go [hunk 2]
```diff
@@ -656,24 +663,36 @@ func (s *Service) lateBlockTasks(ctx context.Context) {
 		return
 	}
 
-	s.headLock.RLock()
-	headBlock, err := s.headBlock()
-	if err != nil {
+	if headState.Version() >= version.EPBS {
```
Comment (terencechain) [inline beacon-chain/blockchain/process_block.go:666]: Don't you want to process `headState` up to the current slot? What if we've already passed the ePBS fork, but the head state is still on the Electra version

File: beacon-chain/blockchain/process_block.go (hunk 1 above)
Comment (potuz) [reply beacon-chain/blockchain/process_block.go:494]: oh that's a mistake actually, from 3 to 4 would have been fine :P

File: beacon-chain/blockchain/process_block.go (hunk 2 above)
Comment (potuz) [reply beacon-chain/blockchain/process_block.go:666]: If the head state is still in Electra then the headBlock will be fine to send the FCU arguments isn't it?

**Pull request merged**